### ISSUE TITLE
is there a build config string, that will turn off all asserts into no-op?

### ISSUE THREAD

**Issue by mingwugmail** [gedaiu/fluent-asserts#94]
Hi,

I use this library not just in unittest, but also as a replacement for the D built-in `assert` in the normal code.  But I want to turn off these Asserts in release build, I'm wondering is there a build config string, that will turn off all asserts into no-op?

If currently there is none, can we add such build config?

Thanks.

**Comment by gedaiu** [comment]
Hmm...  I think we can do this, but then you will have calls to empty functions, hoping that the compiler / optimiser will remove those calls, but I can not guarantee that will happen.

**Comment by mingwugmail** [comment]
I've done some code like this:

```
version (ASSERT) {
public import fluent.asserts;
} else {
  pragma(msg, "NOTE: using dummy Assert!");
class Assert {
  public static void cmp(X, Y)(X x, Y y, string msg="") {}
  alias greaterThan = cmp;
  alias lessThan = cmp;
  alias equal = cmp;
  public static void approximately(X, Y, Z)(X x, Y y, Z=0) {}
}
}

```

**Comment by gedaiu** [comment]
Yes, this is what I was talking about. With this solution you will still have this function call in your code. It can be done. Are you using the last release v13.x.x or master?

**Comment by mingwugmail** [comment]
last release v0.13.0.

**Comment by gedaiu** [comment]
There will be a config for this in the next major release. Thanks!
